package.json

```diff
@@ -24,7 +24,8 @@
     "commander": "^2.11.0",
     "dockerode": "^2.5.0",
     "fs-extra": "^4.0.2",
-    "grpc": "1.10.1",
+    "grpc": "^1.12.2",
+    "iroha-lib": "^0.1.1",
```

Comment:
1. what's the purpose of add npx?
2. Is grpc > 1.12 MUST be used for iroha. If not, please keep the version as 1.10.1 (we found the new version of grpc is some how incompatible with fabric-sdk )
3. This package is for common use, so please don't add iroha's dependencies here.

Reply:
1. According to the philosophy of NPM all apps should be self-contained. If I have no globally installed ESLint or version of my installed ESLint another than specified in the package for some reason (e.g. in a case of CI), `npm test` and `npm run-script lint` may have strange behavior or even don't work at all.
So I add **npx** as script wrapper to return such functionality.
If you want to I can replace it with directly calling `./node_modules/.bin/eslint .` or even return it to the previous state.
2. grpc mast not be used by Iroha, I returned it to the 1.10.1
3. **IMPORTANT!** We definitely need to add some mechanism to fix versions of used connector packages for different ledgers in order to ensure the stability of the current version of Caliper.
Exactly like we did use docker images and hashes/tags.
I think just description in Readme not enough to solve this issue, therefore I decided to fix version in the common package.json until the future improvements.
I know this not the best solution, but it is something.
If you have any ideas how to gracefully solve this problem, it would be nice!

By the way, is **grpc** in package.json not a ledger-related dependency too?